deep_qa/run.py

```diff
@@ -83,7 +96,9 @@ def run_model(param_path: str, model_class=None):
         handler.setLevel(logging.INFO)
         handler.setFormatter(logging.Formatter('%(asctime)s - %(levelname)s - %(name)s - %(message)s'))
         logging.getLogger().addHandler(handler)
-        shutil.copyfile(param_path, log_dir + "_model_params.json")
+        serialisation_params = deepcopy(params).as_dict()
```

Comment:
This is going to log all of the parameters, won't it?  Maybe adding a parameter to `as_dict()` that tells it to be quiet would be a good idea.